components/brave_rewards/common/features.h

```diff
@@ -13,6 +13,7 @@ struct Feature;
 namespace brave_rewards {
 namespace features {
 
+extern const base::Feature kGeminiFeature;
```

Comment:
you need a buildflag everywhere you reference this

Reply:
added: https://github.com/brave/brave-core/pull/8566/files#diff-6523d69aa2b9e67249a5f71bd635b1effca2a09722972e4fd4fe71fd55c3da48R13